tripal_chado/src/api/tripal_chado.phylotree.api.php

```diff
@@ -998,26 +997,25 @@ function chado_phylogeny_lookup_organism_by_name($name, $schema_name = 'chado')
  *
  * @ingroup tripal_phylotree_api
  */
-function chado_phylogeny_get_node_types_vocab($options, $schema_name = 'chado') {
+function chado_phylogeny_get_node_types_vocab($chado, $options, $schema_name = 'chado') {
```

Comment:
This is not backwards compatible and we have not added deprecation messages indicating the change :see_no_evil:

If this change was made because of the default of schema name for chado then we can either update the importer to pass in the schema name or we can add a section here that if the schema name is not passed in then do not set it or grab the set default schema name.

```suggestion
function chado_phylogeny_get_node_types_vocab($options, $schema_name = 'chado') {
  $chado = \Drupal::service('tripal_chado.database');
  $chado->setSchemaName($schema_name);
```